server.test.js

```diff
@@ -2,8 +2,13 @@ const mongoose = require('mongoose');
 const chai = require('chai');
 const { expect } = chai;
 const sinon = require('sinon');
+const chaiHTTP = require('chai-http');
 
 const Game = require('./models');
+const server = require('./server');
+chai.use(chaiHTTP);
+
+
 
```

Comment:
Try to limit empty lines.  It's definitely okay to separate chunks of code with an empty line, but if you can keep it to one empty line, it helps to keep everything easy to read and find.